### PR TITLE
added ID to config.json files

### PR DESCRIPTION
**Pull request by jkl071**

### CODE REVIEW

**Comment by vsoch** [comment]
Hey looks like there are some conflicts, if you want to resolve them and update the PR. Likely this happened because you did not rebase with the upstream master branch before making edits.

**Comment by jkl071** [comment]
I have pulled changes from upstream master

**Comment by vsoch** [comment]
Great job! I'll be back at my computer soon to merge!

File: tower_of_london/config.json [hunk 1]
```diff
@@ -6,8 +6,10 @@
                 "style.css",
                 "static/js/jspsych/custom_plugins/jspsych-single-stim-button.js"
                ],
-        "tag": "tower_of_london",
-        "cognitive_atlas_task_id": "",
+
+        "tag": "tol",
```
Comment (vsoch) [inline tower_of_london/config.json]: This should be "tower_of_london" and not "tol" - the task tag must be equivalent to the folder name

File: tower_of_london_imagine/config.json [hunk 2]
```diff
@@ -6,8 +6,10 @@
                 "style.css",
                 "static/js/jspsych/custom_plugins/jspsych-single-stim-button.js"
                ],
-        "tag": "tower_of_london_imagine",
-        "cognitive_atlas_task_id": "",
+
+        "tag": "tol",
```
Comment (vsoch) [inline tower_of_london_imagine/config.json]: same here

File: tower_of_london/config.json [hunk 3]
```diff
@@ -6,8 +6,10 @@
                 "style.css",
                 "static/js/jspsych/custom_plugins/jspsych-single-stim-button.js"
                ],
-        "tag": "tower_of_london",
-        "cognitive_atlas_task_id": "",
+
+        "tag": “tower_of_london”,
```
Comment (vsoch) [inline tower_of_london/config.json]: this has some strange character that isn't a " - that's why it looks red.

File: tower_of_london_imagine/config.json [hunk 4]
```diff
@@ -6,8 +6,10 @@
                 "style.css",
                 "static/js/jspsych/custom_plugins/jspsych-single-stim-button.js"
                ],
-        "tag": "tower_of_london_imagine",
-        "cognitive_atlas_task_id": "",
+
+        "tag": “tower_of_london_imagine”,
+        "cognitive_atlas_task_id": "trm_5696bcf1b5c64",
```
Comment (vsoch) [inline tower_of_london_imagine/config.json]: ditto here

**Comment by vsoch** [comment]
I'll fix them up :)

**Pull request merged**